src/main/resources/static/js/registration.js

```diff
@@ -1,15 +1,15 @@
-	var firstname;
-	var lastname;
-	var password;
-	var cpassword;
-	var dateofbirth;
-	var mail;
-	var phone;
-	var address;
-	var city;
-	var zipcode;
-	var country;
-	var state;
+var firstname;
+var lastname;
+var password;
+var cpassword;
+var dateofbirth;
```

Comment:
erg minor dingetje, maar probeer consistent te zijn in camelcasing (maar steek er nu vooral geen tijd meer in om aan te passen)